log4j-samples-gradle-metadata/gradlew

```diff
@@ -0,0 +1,251 @@
+#!/bin/sh
```

Comment:
Can we have a single Gradle Wrapper for both projects?

Reply:
To answer this and https://github.com/apache/logging-log4j-samples/pull/279#discussion_r1976651888 (Jar in repo) together:

Both examples are independent projects. There is not realy good way to share a wrapper between two independent projects. What I would propose is to remove it completely (scrips and Jars) from both examples – Android and this one – and configure the Gradle version in the [`gradle/actions/setup-gradle` action instead](https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#build-with-a-specific-gradle-version).

Wdyt?

Reply:
> What I would propose is to remove it completely (scrips and Jars) from both examples – Android and this one – and configure the Gradle version in the [`gradle/actions/setup-gradle` action instead](https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#build-with-a-specific-gradle-version).

@jjohannes, doesn't this mean that I won't be able to build these myself if I don't have Gradle installed on my system?

Reply:
Yes, that would be the drawback. Not sure how important that is for these example projects if they are mainly used for integration testing (?).

What we can do as compromise is to move the wrapper to the root of the repo. And configure the action to use that.
A user would then need to use a relative path `../gradlew` from inside the sample folder. And IntelliJ won't find the wrapper/version when you open one of the sample projects. But that's maybe acceptable.

Reply:
I did the following:
- Moved the wrapper from the Android example to the root (and not add a second one)
- Add a `settings.gradle` to the root as a "umbrella build" that includes all Gradle samples (android and the new one) into a composed build. When you open the root in IDEA, it will now ask you if you want to open it as Maven project (for the Maven samples) or Gradle project (for the Gradle samples). From the command line, you can also run samples from the root or go into the individual sample projects.

This feels like a good solution to me. Let me know what you think.